Include message triangle on all pages

yubiuser opened the pull request:
- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/AdminLTE/blob/master/CONTRIBUTING.md), as well as this entire template.
- [x] I have made only one major change in my proposed changes.
- [ ] I have commented my proposed changes within the code.
- [x] I have tested my proposed changes.
- [x] I am willing to help maintain this change if there are issues with it later.
- [x] I give this submission freely and claim no ownership.
- [x] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
- [x] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))
- [x] I have Signed Off all commits. ([`git commit --signoff`](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---signoff))

---

**What does this PR aim to accomplish?:**

PR https://github.com/pi-hole/AdminLTE/pull/2133 moved the function `checkMessages()` to `utils.js`. All pages that should call that function (hence show the message triangle) need to source `utils.js`
This PR add this to some pages that have been forgotten.

yubiuser committed:
Include message triangle on all pages

Signed-off-by: Christian König <ckoenig@posteo.de>

rdwebdesign reviewed (commented):
If **all pages** will use this script, I think we should add it to `header.php` and remove it from all other files.

What do you think?

yubiuser committed:
Source utils.js in header.php

Signed-off-by: Christian König <ckoenig@posteo.de>

yubiuser commented:
All pages require `scripts/pi-hole/php/header.php`. So I moved the call to `utils.js` (which includes `checkMessages()`) to `header.php` and removed all the individual calls to `utils.js`

rdwebdesign commented on scripts/pi-hole/php/header.php:
```diff
@@ -662,6 +662,9 @@ function pidofFTL()
         <!-- Main content -->
         <section class="content">
+          
+<script src="scripts/pi-hole/js/utils.js?v=<?=$cacheVer?>"></script>
```
I saw you placed the script near the end of the file.

I just think it's easier to maintain if all scripts are on the same place, but sometime it's not possible.

Why didn't you add this before the `</head>` tag (line 232)? 
Is there a reason for that?

yubiuser replied:
Because I didn't see all that...

yubiuser committed:
I was blind but now I see!

Signed-off-by: Christian König <ckoenig@posteo.de>

yubiuser commented:
Closing and re-opening to trigger checks - somehow they are stuck

rdwebdesign commented:
Github problems: 
https://www.githubstatus.com/
